### ISSUE TITLE
Can't select element to relate to from modal

### ISSUE THREAD

**Issue by svale** [verbb/many-to-many#24]
Hi! 

When trying to add a (reverse) relationship from a Many to Many field, the element select modal opens up empty (no elements to select from).

Probably because of changes introduces in Craft 3.1, ref: https://craftcms.com/guides/updating-plugins-for-craft-3-1#update-element-source-key-references

So it seems this is easily fixed by changing 
```
'section' => $relatedSection->id
``` 
to
```
'section' => $relatedSection->uid
```
in `craft-manytomany/src/fields/ManyToManyField.php`> `getInputHtml()`

Thank you for providing a nice plugin!

---
Craft 3.1.15 with project.yaml

**Comment by alexrubin** [comment]
I have this issue as well.

**Comment by bertheyman** [comment]
@svale : this was recently fixed, so you might want to try again if it's working for you now :)

**Comment by matthisamoto** [comment]
I am also having this issue currently on M2M v1.0.2.2 attempting to inversely relate an entry to an entry directly, not through categories but by using an Entries field as the Associated Field.

I just updated to Craft 3.3 and I do not know if this was working/not working as intended before I did that.

**Comment by engram-design** [comment]
Fixed in [2.0.0](https://github.com/verbb/many-to-many/releases/tag/2.0.0)
